egs2/sdsv21/spk1/local/generate_task_tokens.py

```diff
@@ -0,0 +1,26 @@
+import sys
```

Comment:
I think this file can be removed. It's a temporary feature and is not needed here.